### PR TITLE
Add "prefer-const" ESLint rule and run lint:fix

### PR DESCRIPTION
**Pull request by lucasrmendonca**
### Description, Motivation and Context
The [prefer-const ESLint rule](https://eslint.org/docs/latest/rules/prefer-const) detects variables that are not reassigned and suggests them to be declared as "const" instead;

### What kind of change does this PR introduce?

- [x] Refactoring

### Checklist:
- [x] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [x] Overall test coverage is not decreased.
- [x] All new and existing tests passed.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "no-constant-condition": ["error"],
         "no-else-return": ["error"],
         "object-curly-spacing": ["warn", "always"],
+        "prefer-const": ["error"],
```
Comment (KSDaemon) [inline .eslintrc.json:29]: Do you think this should be error rather than just warning?

**Review by KSDaemon — COMMENTED** [review]
👍🏻 Another one question left

File: .eslintrc.json (hunk 1 above)
Comment (lucasrmendonca) [reply .eslintrc.json:29]: A warning is better, I will update it

Comment (KSDaemon) [reply .eslintrc.json]: 👍🏻

**Comment by KSDaemon** [comment]
@lucasrmendonca Please merge latest changes from dev and resolve the conflicts. Without it I won't be able to merge.

**Comment by KSDaemon** [comment]
👍🏻  Great! Merging!

**Pull request merged**